src/pages/DegreeToggle.js

```diff
@@ -0,0 +1,33 @@
+import React from "react";
+
+const DegreeToggle = ({updateForecastDegree, degreeType}) => {
+  return (
+    <div>
+      <input
+        type="radio"
+        name="degree-type"
+        className="unitCheck"
+        id="celsius"
+        value="celsius"
+        onChange={updateForecastDegree}
+        checked={degreeType === "celsius"}
+      />
+      <label for="celsius">Celsius</label>
+
+      <input
+        type="radio"
+        name="degree-type"
+        className="unitCheck"
+        id="fahrenheit"
+        value="imperial"
+        // onChange={event => updateForecastDegree(event.target.value)}
+        // onChange={props.updateForecastDegree}
+        // checked={props.degreeType === "fahrenheit"}
+        style={{ marginLeft: 30 }}
```

Comment:
Comments are great on one hand but on the other hand, you don't want to ship commented out code. It's best practice to remove all comments that aren't sending a message to your fellows. If it's code that doesn't affect your component/feature, you're better off removing it or storing it in your stash in git. ( we should discuss how to do this as well)